_includes/header.html

```diff
@@ -24,7 +24,7 @@
         <ul class="usa-nav__secondary-links">
           <li class="usa-nav__secondary-item js-glossary-toggle">
             <a href="javascript:void(0)" class="icon-glossary">
-              <svg class="display-none desktop:display-block" width=" 14" height="16" viewBox="0 0 14 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <svg class="display-none desktop:display-block" aria-hidden="true" role="img" alt="Glossary" width=" 14" height="16" viewBox="0 0 14 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
If we're hiding it via `aria-hidden="true"`, I don't think we need the `role` or `alt` attributes.